lib/agent/agent-glue.c

```diff
@@ -98,7 +102,18 @@ frida_agent_environment_init (void)
   gum_init ();
   gum_script_backend_get_type (); /* Warm up */
   frida_error_quark (); /* Initialize early so GDBus will pick it up */
+#ifdef HAVE_LINUX
+  frida_libdl_prevent_unload ();
+#endif
+}
+
+#ifdef HAVE_LINUX
+void
+frida_libdl_prevent_unload ()
```

Comment:
Should be `(void)` instead of `()`, which in C means "any number of arguments", unlike in C++ where it means 0 arguments. Confusing :)